examples/webgl_interactive_cubes_gpu.html

```diff
@@ -202,46 +242,45 @@
 
 			function pick() {
 
-				//render the picking scene off-screen
-
+				// render the picking scene off-screen
 				// set the view offset to represent just a single pixel under the mouse
-
-				camera.setViewOffset( renderer.domElement.width, renderer.domElement.height, pointer.x * window.devicePixelRatio | 0, pointer.y * window.devicePixelRatio | 0, 1, 1 );
+				const dpr = window.devicePixelRatio;
+				camera.setViewOffset(
+					renderer.domElement.width, renderer.domElement.height,
+					Math.floor( pointer.x * dpr ), Math.floor( pointer.y * dpr ),
+					1, 1
+				);
 
 				// render the scene
-
 				renderer.setRenderTarget( pickingTexture );
+
+				// clear the background to - 1 meaning no item was hit
+				clearColor.setRGB( - 1, - 1, - 1 );
+				renderer.setClearColor( clearColor );
+				renderer.clear();
+
+				renderer.autoClear = false;
 				renderer.render( pickingScene, camera );
+				renderer.autoClear = true;
```

Comment:
Autoclear must be disabled since the default approach to clearing a buffer will fail on integer targets. I will look into this another time to see if this can be improved.

Reply:
Looks like I made a mistake - no need to disable. Fixed in later commit.